test/integration/api/submissions.js

```diff
@@ -2235,17 +2235,34 @@ one,h,/data/h,2000-01-01T00:06,2000-01-01T00:07,-5,-6,,ee,ff
                 done();
               })))))));
 
-    it('should not log the action in the audit log', testService((service) =>
```

Comment:
Ah I think I added this test, in #382. The test description is not great, but I think it's still a useful test. It's trying to test that when you download draft submissions, a `form.submission.export` audit log entry is not created.

Reply:
oh i see